src/content/en/fundamentals/accessibility/04-semantics-aria/semaria-five.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: shared/narrow
+title: "ARIA Relationships"
+description: "Using ARIA relationship attributes"
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 5
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+Earlier we discussed `aria-labelledby`, which is an example of a *relationship attribute*. A relationship attribute creates a semantic relationship between elements on the page regardless of their DOM relationship. In the case of `aria-labelledby`, that relationship is "this element is labelled by that element".
+
+The ARIA specification lists <a href="https://www.w3.org/TR/wai-aria/states_and_properties#attrs_relationships" target="_blank">eight relationship attributes</a>. Six of these, `aria-activedescendant`, `aria-controls`, `aria-describedby`, `aria-flowto`, `aria-labelledby`, and `aria-owns`, take a reference to one or more elements to create a new link between elements on the page. The difference in each case is what that link means and how it is presented to users.
+
+`aria-owns` is one of the most widely used ARIA relationships. This attribute allows us to tell assistive technology that an element that is separate in the DOM should be treated as a child of the current element, or to rearrange existing child elements into a different order. For example, if a pop-up sub-menu is visually positioned near its parent menu, but cannot be a DOM child of its parent because it would affect the visual presentation, you can use `aria-owns` to present the sub-menu as a child of the parent menu to a screen reader.
```

Comment:
```html
<div role="menu">
  <div role="menuitem"
       aria-haspopup="true">
    New
  </div>
  <div>
  </div>
  <!-- more items… -->
</div> <!-- menu -->
<div role="menu" id="submenu">
  <div role="menuitem">
    Document
  </div>
  <!-- more items… -->
</div> <!-- submenu -->
```

BTW, all our slides are in https://drive.google.com/corp/drive/u/0/folders/0BxoQ5N_Uc9G-fm9GRy1MNlczZU1hdjYtTFlRUHlMSV9zYkJHVURrTHlnNzhfSUtqUWIwOEE and many of them contain sample code.